File: ios/RNAltheaAdyenCheckout.m

#if __has_include("RCTBridgeModule.h")
#import "RCTBridgeModule.h"
#else
#import <React/RCTBridgeModule.h>
#endif
  
@interface RCT_EXTERN_MODULE(RNAltheaAdyenCheckout, NSObject)

RCT_EXTERN_METHOD(getPaymentToken)

@end

File: ios/RNAltheaAdyenCheckout.swift
//
//  RNAltheaAdyenCheckout.swift
//  RNAltheaAdyenCheckout
//
//  Created by Alvin on 14/03/2019.
//  Copyright © 2019 Facebook. All rights reserved.
//

import Foundation
import Adyen

@objc(RNAltheaAdyenCheckout)
class RNAltheaAdyenCheckout: NSObject {
    
    @objc
    func getPaymentToken(
        _ resolve: RCTPromiseResolveBlock,
        rejecter reject: RCTPromiseRejectBlock
        ) -> Void {
        
        resolve("It works!");
    }
}
